include/fi_mem.h

```diff
@@ -62,15 +62,69 @@ static inline void *mem_dup(const void *src, size_t size)
 	return dest;
 }
 
+
+/*
+ * Buffer pool (free stack) template
+ */
+#define FREESTACK_EMPTY	-1
```

Comment:
Oh strange, the Github diff viewer is missing the space but if I look at the raw version it has the space...